doc: remove spurious console code block from arduino_zero doc.

henrikbrixandersen opened the pull request:
Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

henrikbrixandersen committed:
doc: remove spurious console code block from arduino_zero doc.

Remove spurious console code block to fix formatting of the following
zephyr-app-commands directive. While I am here, fix the indentation of
the two zephyr-app-commands directives.

Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

dbkinder commented on boards/arm/arduino_zero/doc/arduino_zero.rst:
```diff
@@ -120,8 +120,6 @@ Flashing
 
-   .. code-block:: console
-
 	.. zephyr-app-commands::
```
You should also outdent this ``.. zephyr-app-commands::`` directive too (align with the T in "To flash")
(the same for the directive on line 97 above)

dbkinder reviewed (changes requested):
needs a body in the commit message.  Also, while you're in there can you fix the indent of the two zephyr-app-commands directive.

henrikbrixandersen commented:
@dbkinder Thanks. I have updated the commit to fix the indentation of the two zephyr-app-commands directives and added a commit message body. Please review.

dbkinder reviewed (approved):
Thanks
